tendrl/node_agent/manager/__init__.py

```diff
@@ -90,7 +95,13 @@ def main():
     if NS.config.data.get("with_internal_profiling", False):
         from tendrl.commons import profiler
         profiler.start()
-
+    # Initialize alert count
+    try:
```

Comment:
This code needs  to be in node_agent/node_sync/__init__.py 

Not here

Reply:
ok